src/components/LineSeries.tsx

```diff
@@ -54,16 +61,36 @@ function LineSeriesRender({ data, lineStyle, label }: LineSeriesRenderProps) {
     const lineGenerator = line<{ x: number; y: number }>()
       .x((d) => d.x)
       .y((d) => d.y);
-    return lineGenerator(series);
-  }, [xScale, yScale, data]);
+
+    // Show markers
+    const markers = !displayMarker
+      ? null
+      : series.map(({ x, y }, i) => (
+          <circle
+            // eslint-disable-next-line react/no-array-index-key
+            key={`markers-${i}`}
+            cx={x}
+            cy={y}
+            r="3px"
+            fill={color}
```

Comment:
this should be a 'style' property so that everything can be customised using any of the SVG style property. By default it could be style: {fill: color}

Reply:
There's any reason to change only the markers colors?